misc/docerina/src/test/java/org/ballerinalang/docgen/tests/GenDocsForBalaTest.java

```diff
@@ -62,6 +62,8 @@ public void generatingDocsForBalaTest() throws IOException {
         Assert.assertTrue(sfModuleApiDocsJsonAsString.contains("## Module Overview\\n\\nModule.md content."),
                 "Module.md content is missing");
         Assert.assertTrue(sfModuleApiDocsJsonAsString.contains("Block"), "Block type is missing");
+        Assert.assertTrue(sfModuleApiDocsJsonAsString.contains("\"summary\":\"Module.md content.\""),
```

Comment:
This test confirms it will not add links in brackets?

Reply:
Links are aadded in brackets because of the parsing by(https://github.com/ballerina-platform/ballerina-lang/blob/5251d29fe6ddbdc9b022ea4d8738eee723e94c45/misc/docerina/src/main/java/org/ballerinalang/docgen/docs/utils/BallerinaDocUtils.java#L178)

With the new implementation, we are getting the content as it is in the Module.md file.

Reply:
Added the changes.